lessons/5-github-basics/slides.qmd

```diff
@@ -75,15 +59,34 @@ Figure 2 from [@bryan2018]
 -   Push local changes to GitHub
 -   Pull GitHub changes to local
 
+## Ignoring files {.smaller}
```

Comment:
I think this will be a good complement to introducing .gitignore in the git lesson :)